worlds/baylands.world

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" ?>
+<sdf version="1.5">
+  <world name="default">
+    <scene>
+      <sky>
+        <clouds>
+          <speed>12</speed>
+        </clouds>
+      </sky>
+    </scene>
+    <!-- A global light source -->
+    <include>
+      <uri>model://sun</uri>
+    </include>
+
```

Comment:
Could you define the gps origin as defined in https://github.com/PX4/sitl_gazebo/blob/master/worlds/sonoma_raceway.world#L12 ?